scripts/npc/npc_abilities_custom.txt

```diff
@@ -8453,7 +8454,7 @@
 	"ability_priest_angelicelemental"
 	{
 
-		--If possible can we tint particles that are called upon?
```

Comment:
using -- for comments inside K/V files is illegal. I believe // works but not sure if it does inside the dicts